libvast/src/system/exporter.cpp

```diff
@@ -175,9 +172,10 @@ behavior exporter(stateful_actor<exporter_state>* self, expression expr,
   self->set_exit_handler(
     [=](const exit_msg& msg) {
       VAST_DEBUG(self, "received exit from", msg.source, "with reason:", msg.reason);
-      self->send<message_priority::high>(self->state.index, self->state.id, 0);
-      self->send(self->state.sink, sys_atom::value, delete_atom::value);
-      self->send_exit(self->state.sink, msg.reason);
+      auto& st = self->state;
+      self->send<message_priority::high>(st.index, st.id, 0);
```

Comment:
Can you add a comment what this does? I think we "abuse" some of the message handlers at the INDEX where 0 means "abort" or "done". Since this is not clear in this context, it warrants a comment.